packages/fxa-shared/test/lib/logger.ts

```diff
@@ -0,0 +1,116 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { assert } from 'chai';
+import Logger from '../../lib/logger';
+import sinon = require('sinon');
+
+describe('lib/logger', () => {
+  let logger: Logger;
+
+  describe('constructor', () => {
+    it('should create logger', () => {
+      sinon.spy(window.console, 'info');
+
+      const information = 'Hello Firefox!';
+
+      logger = new Logger();
+      logger.info(information);
+
+      assert.isTrue(
+        (window.console.info as sinon.SinonSpy).calledOnceWith(information)
```

Comment:
If we use `sinon.assert` then it'd show the diff between the actual and expected on failure.  That's more helpful than "found false when expecting true" or whatever the message is.

Reply:
(Of course you could use equal or deepequal from chai to do the same thing.)